rfc/src/rfcs/0003-cover-statement.md

```diff
@@ -85,7 +85,16 @@ fn foo() {
 }
 ```
 
-We can consider adding an option that would cause verification to fail if a cover property was unsatisfiable or unreachable, e.g. `--fail-uncoverable`.
+The `--fail-uncoverable` option will allow users to fail the verification if a cover property is unsatisfiable or unreachable.
+This option will be integrated within the framework of [Global Conditions](https://model-checking.github.io/kani/rfc/rfcs/0007-global-conditions.html), which is used to define properties that depend on other properties.
+
+Using the `--fail-uncoverable` option will enable the global condition with name `fail_uncoverable`.
+Following the format for global conditions, the outcome will be one of the following:
+ 1. `` - fail_uncoverable: FAILURE (encountered one or more cover statements which were not satisfied)``
```

Comment:
I like your use of "expected" in `should_panic` below; IMHO using it here would make this clearer.

```suggestion
 1. `` - fail_uncoverable: FAILURE (expected to reach all cover statements, but one or more were not reached)``
```

Reply:
Yes, much better. Thank you!